XenAdmin/Dialogs/ManageCPUGroupDialog.Designer.cs

```diff
@@ -0,0 +1,300 @@
+            this.buttonCancel.Size = new System.Drawing.Size(75, 23);
+            this.buttonCancel.TabIndex = 69;
+            this.buttonCancel.Text = "Close";
+            this.buttonCancel.UseVisualStyleBackColor = true;
+            this.buttonCancel.Click += new System.EventHandler(this.buttonCancel_Click);
+            // 
+            // NameColumn
+            // 
+            this.NameColumn.AutoSizeMode = System.Windows.Forms.DataGridViewAutoSizeColumnMode.None;
+            this.NameColumn.HeaderText = "Name";
+            this.NameColumn.MinimumWidth = 30;
+            this.NameColumn.Name = "NameColumn";
+            this.NameColumn.ReadOnly = true;
+            this.NameColumn.Width = 200;
+            // 
+            // DescriptionColumn
+            // 
+            this.DescriptionColumn.AutoSizeMode = System.Windows.Forms.DataGridViewAutoSizeColumnMode.Fill;
+            this.DescriptionColumn.HeaderText = "Description";
+            this.DescriptionColumn.MinimumWidth = 30;
+            this.DescriptionColumn.Name = "DescriptionColumn";
+            this.DescriptionColumn.ReadOnly = true;
+            // 
+            // CPUCoresColumn
+            // 
+            this.CPUCoresColumn.AutoSizeMode = System.Windows.Forms.DataGridViewAutoSizeColumnMode.AllCells;
+            this.CPUCoresColumn.HeaderText = "#CPU Cores";
+            this.CPUCoresColumn.MinimumWidth = 30;
+            this.CPUCoresColumn.Name = "CPUCoresColumn";
+            this.CPUCoresColumn.ReadOnly = true;
+            this.CPUCoresColumn.Width = 95;
+            // 
+            // ManageCPUGroupDialog
+            // 
+            this.AutoScaleDimensions = new System.Drawing.SizeF(7F, 15F);
+            this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
+            this.ClientSize = new System.Drawing.Size(721, 428);
+            this.Controls.Add(this.buttonCancel);
+            this.Controls.Add(this.tableLayoutPanel1);
+            this.Controls.Add(this.panel1);
+            this.Font = new System.Drawing.Font("Segoe UI", 9F);
+            this.Name = "ManageCPUGroupDialog";
+            this.Text = "Manage CPU Group";
+            this.panel1.ResumeLayout(false);
+            this.toolStrip1.ResumeLayout(false);
+            this.toolStrip1.PerformLayout();
+            this.tableLayoutPanel1.ResumeLayout(false);
+            this.tableLayoutPanel1.PerformLayout();
+            ((System.ComponentModel.ISupportInitialize)(this.dataGridViewCPUGroups)).EndInit();
+            this.tableLayoutPanel2.ResumeLayout(false);
+            this.ResumeLayout(false);
+
+        }
+
+        #endregion
+        private System.Windows.Forms.Panel panel1;
+        private System.Windows.Forms.ToolStrip toolStrip1;
+        private System.Windows.Forms.ToolStripButton toolStripButtonNew;
+        private System.Windows.Forms.ToolStripButton toolStripButtonDelete;
+        private System.Windows.Forms.ToolStripButton toolStripButtonEddit;

```

Comment:
"toolStripButtonEdit"
Typos in variable names don't cause problems to the UI or the compiler, but they may make trouble for a developer searching the solution by text.

Reply:
Thanks, I have fixed it.